resources/assets/components/InboxItem/index.js

```diff
@@ -30,11 +46,11 @@ class InboxItem extends React.Component {
           <h4>Why Statement</h4>
           <p>{post['signup']['why_participated']}</p>
           <ul className="form-actions -inline">
-            <li> <input className="button" value="Accepted"/></li>
+            <li> <input className="button" value="Accepted" onClick={this.onAcceptClick}/></li>
             <li> <input className="button -secondary" value="Rejected"/></li>
-            <br/>
-            <li> <input className="button -tertiary" value="Delete"/></li>
+            <li> <input className="button -tertiary" id="delete_post" value="Delete"/></li>
```

Comment:
Hmm, okay. I'm surprised the button is getting that extra padding. I think it's okay to merge as a quick-fix, but let's at the very least replace this with a more descriptive classname (maybe `.left-aligned`).